spec/api/holdings_api_spec.rb

```diff
@@ -223,28 +220,48 @@ def parse_response(...)
       load_test_data(ht1, ht2, holding)
 
       # as set up above, upenn's holdings (empty enumchron) matches on ht1 both ht2
-      response = parse_response("item_access", v: 1, organization: holding.organization, item_id: ht1.item_id)
+      response = parse_response("item_access", organization: holding.organization, item_id: ht1.item_id)
       expect(response["copy_count"]).to eq 1
-      response = parse_response("item_access", v: 1, organization: holding.organization, item_id: ht2.item_id)
+      response = parse_response("item_access", organization: holding.organization, item_id: ht2.item_id)
       expect(response["copy_count"]).to eq 1
     end
+
+    it "brlm_count indicates number of BRT/LM holdings by organization" do
+      load_test_data(
+        htitem_1,
+        build(:holding, organization: "umich", ocn: ocn)
+      )
+
+      # brlm_count if given &constraint=brlm, default 0
+      response = parse_response("item_access", item_id: item_id_1, organization: "umich", constraint: "brlm")
```

Comment:
You can get rid of the three instances of `constraint: "brlm"` from the API call.

Reply:
Yep. Good catch. Removed & updated comments.